crates/egui_demo_app/src/apps/custom3d_wgpu.rs

```diff
@@ -47,13 +47,13 @@ impl Custom3d {
             layout: Some(&pipeline_layout),
             vertex: wgpu::VertexState {
                 module: &shader,
-                entry_point: "vs_main",
+                entry_point: Some("vs_main"),
```

Comment:
Let's use that new feature and just pass in `None` because why not :)
(works here but not inside of egui-wgpu renderer.rs)